content/patterns/disclosure/examples/disclosure-navigation.html

```diff
@@ -20,177 +20,169 @@
-            <h3 id="mythical-page-heading">Mythical University</h3>
+          <div id="mythical-page-content" class="disclosure-page-content" tabindex="-1" role="region" aria-label="架空大学のサンプルページコンテンツ">
+            <h3 id="mythical-page-heading">架空大学</h3>
             <p>
-              Sample content section.
-              Activating a link above will update and navigate to this region.
+              サンプルコンテンツセクション。
+              上記のリンクを選択すると、この領域が更新されてナビゲートされます。
             </p>
           </div>
         </div>
         <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ol>
-          <li>Since this set of disclosure buttons controls visibility of navigation links for the mythical university web site, the list that contains them is wrapped in a navigation landmark named <q>Mythical University</q>.</li>
+          <li>この一連のディスクロージャーボタンは架空大学ウェブサイトのナビゲーションリンクの表示を制御するため、それらを含むリストは<q>架空大学</q>という名前のナビゲーションランドマークでラップされています。</li>
           <li>
-            The semantics of the list structure communicates the hierarchy of the navigation system to assistive technology users.
-            The top-level set of three buttons is in a list of three items, and the set of links controlled by each button is contained within a list nested inside the item with the controlling button.
+            リスト構造のセマンティクスは、支援技術の利用者にナビゲーションシステムの階層を伝えます。トップレベルの 3 つのボタンのセットは 3 項目からなるリストに含まれ、各ボタンによって制御されるリンクのセットは、制御ボタンを含むアイテム内のネストされたリスト内に含まれています。
           </li>
           <li>
-            If a dropdown is open and focus is inside the navigation region, pressing <kbd>Esc</kbd> will close the dropdown.
-            Moving focus out of the navigation region also closes an open dropdown.
-            Implementing this <kbd>Esc</kbd> behavior is necessary to meet the WCAG 2.1
-            <a href="https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html">1.4.13: Content on Hover or Focus</a>
-            criterion.
+            ドロップダウンが開かれ、フォーカスがナビゲーション領域内にある場合、<kbd>Esc</kbd> を押すとドロップダウンが閉じます。ナビゲーション領域からフォーカスを外すことでも、開いているドロップダウンは閉じます。この <kbd>Esc</kbd> の振る舞いを実装することは、WCAG 2.1 の <a href="https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html">1.4.13: ホバー又はフォーカスで表示されるコンテンツ</a>基準を満たすために必要です。
           </li>
-          <li>The visual indicator of the show/hide state is created using CSS <code>::after</code> pseudo element border styles so the caret is reliably rendered in high contrast mode of operating systems and browsers.</li>
+          <li>表示/非表示の状態の視覚的インジケーターは、オペレーティングシステム及びブラウザーのハイコントラストモードでキャレットが確実にレンダリングされるよう、CSS の <code>:after</code> 疑似要素のボーダースタイルを使用して作成されています。</li>
           <li>
-            Optional navigation keys (Arrow keys, <kbd>Home</kbd>, and <kbd>End</kbd>):
+            オプショナルなナビゲーションキー（矢印キー及び　<kbd>Home</kbd>、並びに　<kbd>End</kbd>）:
```

Comment:
h2 見出し「使用例のオプション」のチェックボックスのところでは「**オプションの**矢印キー及び Home、並びに End のサポートを含める」とあるので、それに合わせて「オプションの」にするのはいかがでしょうか？

```suggestion
            オプションのナビゲーションキー（矢印キー及び　 <kbd>Home</kbd>、並びに <kbd>End</kbd>）:
```